### ISSUE TITLE
Speed, the speed when capturing the screen is quite slow.

### ISSUE THREAD

**Issue by Leah9** [Leah9/screengrab#6]
Improve the speed of the capture process.

It currently takes around one second per page and is fine for small documents but it could be faster than it is.

An option to limit the speed may also be required to allow pages to load before capture (GUI option).

**Comment by ArielMAJ** [comment]
From what I tested over here, it seems the slowdown is mainly from the pyautogui.press('down') function call. Everything else is almost instantaneous compared to it. 

After reading the implementation of the library (I couldn't find good results on google for some reason), i found out that the press function has a parameter called "_pause". It seems to be boolean and true by default. When I set it to false to loop goes a lot faster.

I just am not sure if it is a good idea to do that because sometimes that "_" might mean it's an internal parameter we shouldn't be touching. I couldn't find much results about it on my rapid search sadly.

This could be a solution if you want to use it.
